test/common/utils/validations.unit.spec.js

```diff
@@ -195,16 +196,16 @@ describe('Validations unit tests', () => {
     it('should validate partial range', () => {
       const fromDate = {
         month: {
-          value: '2'
+          value: 2
```

Comment:
PTAL - this makes the test kind of messy (esp. the places where values generated by moment() are converted to strings), but at least it's consistent now and emulates the components more accurately.